bookshelf.js

```diff
@@ -0,0 +1,76 @@
+window.bookshelf = (function() {
+	var buttonNode = document.createElement("button");
+	buttonNode.setAttribute("id", 'fetch-books-btn');
+	var textNode = document.createTextNode("Fetch Books");
+	buttonNode.appendChild(textNode);
+	buttonNode.addEventListener("click", fetchBooks);
+	var body = document.getElementsByTagName("body")[0];
+	body.appendChild(buttonNode);
+
+	//function createBtn() {
+		function fetchBooks() {
+			const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+			fetch(booksJSON).then(response => response.json()).then(processBooks => {
+				var x = document.createElement("UL");
```

Comment:
please call this with a more significant name like `ul` or `unorderedList`, going down it's quite confusing to remember what that `x` is